test/scripts/benchmarks/parse-benchmark-data-spec.js

```diff
@@ -0,0 +1,21 @@
+// Libraries
+const chai = require('chai');
+const assert = chai.assert;
+const { execSync } = require('child_process');
+
+const testCsv = 'test/scripts/benchmarks/fixtures/test-benchmarks.csv';
+
+// Function which executes script and reads in output file to a JS object.
+const runTest = function(benchmarksCsv) {
+  const cmd = 'cat ' + testCsv + ' | ' + 'node ./scripts/benchmarks/parse-benchmarks-data.js 2015 2017';
+  const benchmark = execSync(cmd).toString();
+  return JSON.parse(benchmark);
+};
+
+describe('parse benchmark data', function() {
+  it('should create new measures and ignore duplicate measureIds', () => {
```

Comment:
if we throw an error when a fuzzy matched measure id can't be found, could we reduce this test case down to checking for duplicate measureIds (using a smaller test csv, checking for specific measure ids in output)? this is a hard test case to maintain; if it breaks i'm not sure how I'd debug what went wrong.